playground/outline-toolkit.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @title Outline
+ * @category Toolkit
+ */
+import * as dat from "dat.gui";
+import { AmbientLight, Animator, AssetType, Camera, GLTFResource, Logger, WebGLEngine, WebGLMode } from "oasis-engine";
+import { FramebufferPicker, OrbitControl, OutlineManager } from "oasis-engine-toolkit";
+Logger.enable();
+const gui = new dat.GUI();
+const engine = new WebGLEngine("canvas", { webGLMode: WebGLMode.WebGL1 });
+engine.run();
+engine.canvas.resizeByClientSize();
+
+const scene = engine.sceneManager.activeScene;
+const rootEntity = scene.createRootEntity();
+scene.background.solidColor.set(1, 1, 1, 1);
+
+// camera
+const cameraEntity = rootEntity.createChild("camera_node");
+cameraEntity.transform.setPosition(0, 1.3, 10);
+
+const camera = cameraEntity.addComponent(Camera);
+camera.enableFrustumCulling = false;
+cameraEntity.addComponent(OrbitControl).target.set(0, 1.3, 0);
+
+const outlineManager = cameraEntity.addComponent(OutlineManager);
+gui.add(outlineManager, "size", 1, 6, 0.1);
+gui
+  .addColor(
+    { color: [outlineManager.color.r * 255, outlineManager.color.g * 255, outlineManager.color.b * 255] },
+    "color"
+  )
+  .onChange((v) => {
+    outlineManager.color.set(v[0] / 255, v[1] / 255, v[2] / 255, 1);
+  });
+
```

Comment:
GUI code should be separated from the main logic through encapsulate a function, so that developers can read it more clearly